### DISCUSSION TITLE
Góp ý Code

### DISCUSSION THREAD

**Discussion by Deepblue3883** [doanguyen/lasotuvi#13] [General]
Cảm ơn ad đã chia sẻ. Mình thấy phần mềm của bạn bị lỗi mấy mục, mình xin góp ý để bạn tham khảo. Mình k phải chuyên về lập trình. 
1. An vòng tràng sinh. Sao thai và dưỡng bị nhầm vị trí. 
2. Sai âm dương thuận lý nghịch lý. 
3. Sao Quan Phủ và Quan Phù bị sai chính tả nên bị lẫn
4. Sao Thiên giải, Văn Khúc, Phá Toái, an sai. 
5. Sai Sao chủ mệnh sao chủ thân. ( Phần code mình thấy có vẻ đúng nhưng hiển thị sai
6. Sai Mệnh và Cục 1983. 
7. Sao Thiên Phủ chưa có hiển thị  M V B H. 

Nếu bạn cần tài liệu mình sẽ cung cấp cho bạn nhé. Cảm ơn bạn đã chia sẻ

**Comment by thanhphucvnvn** [comment]
Hi bạn, 
Bạn có thể add Zalo giúp mình được không? : 0847334128 
Trân trọng,

**Reply by thanhphucvnvn** [reply]
Bạn add giúp mình Zalo nhé: Deepblue3883
